src/mongo/commands.ts

```diff
@@ -78,6 +80,44 @@ export class MongoCommands {
 		}
 		return lastCommandOnSameLine || lastCommandBeforePosition || commands[commands.length - 1];
 	}
+
+	public static async createMongoCollection(db: MongoDatabaseNode, explorer: CosmosDBExplorer) {
+		const collectionName = await vscode.window.showInputBox({
+			placeHolder: "Enter name of collection",
```

Comment:
The placeholder between mongo and docdb should be consistent. I like this version better